src/pages/ActiveVotation.tsx

```diff
@@ -123,9 +121,9 @@ const Votation: React.FC<{ votationId: string; backToVotationList: (status: Vota
   useEffect(() => {
     if (winnerResult?.getWinnerOfVotation) {
       const result = winnerResult.getWinnerOfVotation as Alternative[];
-      const newWinners = result.map((a) => {
-        return { id: a.id, text: a.text, votationId: a.votationId, index: a.index };
-      });
+      const newWinners = result.map((a) => a.text);
+
+      //why newWinners.length > winner.length ?!?!
       if (!winners || (winners && newWinners.length > winners.length)) setWinners(newWinners);
```

Comment:
@endrebmedhus do you know why there is a check for `newWinners.lenth > winner.length`

Reply:
I think it was when there was no result-object, sometimes the result were fetched and set to be no winners, before the actual result was stored, and we therefore ended up with "No winners" when there was actually a winner. This may however have changed with the introduction of votation results, so just checking !winner might be enough.